cmd/delete/accessRestriction/accessRestriction.go

```diff
@@ -0,0 +1,96 @@
+package accessRestriction
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/loginradius/lr-cli/api"
+	"github.com/loginradius/lr-cli/cmdutil"
+
+	"github.com/spf13/cobra"
+)
+
+
+type domain struct {
+	BlacklistDomain    string `json:"blacklistdomain"`
+	WhitelistDomain    string `json:"whitelistdomain"`
+}
+
+
+func NewaccessRestrictionCmd() *cobra.Command {
+	opts := &domain{}
+
+	cmd := &cobra.Command{
+		Use:   "access-restriction",
+		Short: "Deletes Whitelisted/Blacklisted Domains/Emails",
```

Comment:
We can use the argument as `--domain` for white/blacklist a domain
`--email` for white/blacklist an email, as `delete` for to `whitelist` and `add` to `block` right?

Reply:
@mohammed786  from the perspective of admin console , we defined the command here so it will not confuse the customer about the section we defined in our dashboard. 
But yea thats good suggestion ..we will try to minimize the add command and use the arguements